configfiles/6100-httpd.conf

```diff
@@ -126,8 +130,10 @@ filter {
       }
     }
 
-    mutate {
```

Comment:
wanted to clarify this - the "remove_field" mutation should simply skip quietly if the specified field does not exist, so the "if" conditional shouldn't be needed.  But I wanted to check to see if there was some situation where you encountered a problem without the conditional.

Reply:
Hey @philhagen you're right! I noticed that while testing, and I must have forgotten to remove it in the pull request. 😅